docs/other-api/dev.md

```diff
@@ -34,21 +34,21 @@ import {} from "remix";
 Builds your app for production. No need to add `NODE_ENV=production` to the command.
 
 ```sh
-$ remix build
+remix build
```

Comment:
This is outside of the scope of this PR but similar to the `json` snippet above, one could render a `$` as a `::before` pseudo-element here and make the code copy&paste-able as well.